llvm/test/CodeGen/LoongArch/lsx/vec-sext.ll

```diff
@@ -6,10 +6,10 @@ define void @load_sext_2i8_to_2i64(ptr %ptr, ptr %dst) {
 ; CHECK-LABEL: load_sext_2i8_to_2i64:
```

Comment:
What's the relationship between this test and this patch? I suppose you should add a dedicate `shufflevector` test.